### DISCUSSION TITLE
gesp-1-bcqm3017-3018/

### DISCUSSION THREAD

**Discussion by giscus** [lihongzheshuai/lihongzheshuai.github.io#347] [General]
# gesp-1-bcqm3017-3018/

接触输入后，两道简单的读取输入后，按要求输出的变量值的题。基本语法的应用，没什么难度。

https://www.coderli.com/gesp-1-bcqm3017-3018/

**Comment by Niuger** [comment]
2024.9.28打卡 这样3018也该说是输入整数才是呀
